.gitattributes

```diff
@@ -0,0 +1 @@
+testdata/*    -text

```

Comment:
We really want to disable all the CRLF/LF line conversions that git likes to do. This was responsible for much of the pain in the CI test jobs. We want to treat everything inside `testdata/` as binary files (as far as git is concerned) - please don't mess with anything! Check them out so that they are the exact same bytes on disk on Windows & Unix.

Reply:
Where is this .gitattributes file, please? I doesn't seem to be in the GitHub repo.

Reply:
Oh hang on, I'm being silly. You are proposing to ADD this file. Doh!